docs/conf.py

```diff
@@ -0,0 +1,111 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import django
+
+from django.conf import settings
+
+sys.path.insert(0, os.path.abspath('../'))
+sys.path.insert(0, os.path.abspath('../sortinghat'))
+
+settings.configure(
+    SECRET_KEY = 'fake_key',
+    INSTALLED_APPS=[
+        'sortinghat.core',
+        'django.contrib.contenttypes',
+        'django.contrib.admin',
+        'django.contrib.auth'
+    ],
+    RQ_QUEUES = {
+    'default': {
+        'HOST': 'localhost',
+        'PORT': 6379,
+        'ASYNC': True,
+        'DB': 0
+        }
+    },
+    DEFAULT_GRAPHQL_PAGE_SIZE = 2
+)
+django.setup()
```

Comment:
Again, I don't know why we need this.

Reply:
The readthedocs build will leave some warning and some of the developer docs will be missing. Those configs are needed by the build to generate the docs properly.

Reply:
Ok, well, let's keep it this way, then. I guess is because sphynx imports the packages and modules to get the docstrings, classes, etc.